persistence/src/main/resources/db/migration/V1_0__init.sql

```diff
@@ -4,6 +4,11 @@ CREATE TABLE "trafficincident"
 (
     "acquisitiontime" TIMESTAMP WITH TIME ZONE,
     "trafficincidenttype_id" BIGINT,
+    "mediaurl" VARCHAR(255),
```

Comment:
`media_url`

Reply:
I could see a length of 255 being too short in some cases. URLs can get quite long :see_no_evil: